packages/angular-workspace/package.json

```
@@ -25,29 +25,29 @@
     "format": "ng lint --fix"
   },
   "dependencies": {
-    "@angular/animations": "^16.2.12",
-    "@angular/common": "^16.2.12",
-    "@angular/compiler": "^16.2.12",
-    "@angular/core": "^16.2.12",
-    "@angular/forms": "^16.2.12",
-    "@angular/platform-browser": "^16.2.12",
-    "@angular/platform-browser-dynamic": "^16.2.12",
-    "@angular/router": "^16.2.12",
+    "@angular/animations": "^17.3.11",
+    "@angular/common": "^17.3.11",
+    "@angular/compiler": "^17.3.11",
+    "@angular/core": "^17.3.11",
+    "@angular/forms": "^17.3.11",
+    "@angular/platform-browser": "^17.3.11",
+    "@angular/platform-browser-dynamic": "^17.3.11",
+    "@angular/router": "^17.3.11",
     "@ni/nimble-components": "*",
     "@ni/spright-components": "*",
     "rxjs": "^7.3.0",
```

Based on https://v17.angular.io/guide/versions#actively-supported-versions

```suggestion
    "rxjs": "^7.4.0",
```